### PR TITLE
Form.changed_data is not callable fixes #148

### LINKED ISSUE #148: Form changed_data should not be callable
The form stubs have `changed_data` marked as a `Callable[[], List[str]]`, but it's not callable, it's a [property](https://github.com/django/django/blob/master/django/forms/forms.py#L431-L432),  so I think it should just be `List[str]`

### PR DESCRIPTION
**Pull request by jaredlockhart**
When trying to annotate references to `changed_data` I noticed that mypy suggests its type should be `Callable[[BaseForm], List[str]]` but since it's an `@cached_property` it is not callable, so it should be `List[str]`.

### CODE REVIEW

File: django-stubs/forms/forms.pyi [hunk 1]
```diff
@@ -29,6 +29,7 @@ class BaseForm:
     fields: Dict[str, Any] = ...
     renderer: BaseRenderer = ...
     cleaned_data: Any = ...
+    changed_data: List[str]: ...
```
Comment (sobolevn) [inline django-stubs/forms/forms.pyi]: Can we make it a `@propery`? I guess it is the closest match.

**Review by sobolevn — COMMENTED** [review]
Thanks a lot! Good catch.

File: django-stubs/forms/forms.pyi (hunk 1 above)
Comment (jaredlockhart) [reply django-stubs/forms/forms.pyi]: Oh good idea, I changed it 👍

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thanks!

**Comment by jaredlockhart** [comment]
@sobolevn Thanks for merging! When will the next tag be released with this change?

**Comment by sobolevn** [comment]
Sorry, we don't have any release schedule yet. Can you use `master` for now? 

@mkurnikov can you please help us here?

**Pull request merged**